Applets/UI/Onboarding/OnboardingViews/OnBoarding.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  OnBoarding.swift
+//  Applets
+//
+//  Created by シェイミ on 17/02/2023.
+//
+
+
+import SwiftUI
+import CoreGraphics
+
+struct OnBoarding: View {
+    @State private var offset: CGFloat = 0
+    @State private var isEUButtonPressed = false
```

Comment:
This is not actually toggling the isEUCitizen value in the user model, because that has not been merged yet. What it does is turn the buttons into radio buttons, where one it pressed and the other is unpressed

Reply:
I've simplified and refactored a bit.